packages/ui-components/src/components/Chart/Table.js

```diff
@@ -0,0 +1,176 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import MuiTableContainer from '@material-ui/core/TableContainer';
+import MuiTable from '@material-ui/core/Table';
+import MuiTableBody from '@material-ui/core/TableBody';
+import MuiTableCell from '@material-ui/core/TableCell';
+import MuiTableHead from '@material-ui/core/TableHead';
+import MuiTableRow from '@material-ui/core/TableRow';
+import { formatDataValueByType } from '@tupaia/utils';
+import { formatTimestampForChart, getIsTimeSeries, getIsChartData, getNoDataString } from './utils';
+import { SmallAlert } from '../Alert';
+
+const TableContainer = styled(MuiTableContainer)`
+  overflow: auto;
+`;
+
+const StyledTable = styled(MuiTable)`
+  table-layout: fixed;
+  overflow: hidden;
+
+  // table head
+  thead {
+    text-transform: capitalize;
+    border-bottom: 1px solid ${props => props.theme.palette.grey['400']};
+
+    tr {
+      background: none;
+    }
+
+    th {
+      position: relative;
+      height: 7.5rem;
+      text-align: left;
+      border: none;
+      font-weight: 400;
+      vertical-align: bottom;
+    }
+  }
+
+  // table body
+  tbody {
+    th {
+      font-weight: 500;
+    }
+
+    tr {
+      &:nth-of-type(odd) {
```

Comment:
This is a cool css selector I didn't know about!

Reply:
the dark art of css 🧙